kolena/_experimental/object_detection/workflow.py

```diff
@@ -17,7 +17,7 @@
 try:
     from typing import Literal
 except ImportError:
-    from typing_extensions import Literal
+    from typing_extensions import Literal  # type: ignore
```

Comment:
`typing.Literal` was introduced in python 3.8, considering we already dropped python 3.7 support after 1.0, i think we can remove these try catch

Reply:
Will remove